src/styles/blocks/header.scss

```
@@ -0,0 +1,63 @@
+.header {
+  background-color: $white_color;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+
+  &__link-img {
+    display: inline-block;
+    align-items: center;
+    width: $logo-sige;
+    height: $logo-sige;
+    padding: 0;
+  }
+
+  &__list {
+    margin: 0;
+    display: flex;
+    list-style: none;
+    padding-left: 0;
```

```suggestion
    display: flex;
```
Create `reset.scss` and move `margin`, `padding` and `list-style` there (ul, li  selectors)